src/simulators/statevector/chunk/chunk_container.hpp

```diff
@@ -534,6 +154,12 @@ class ChunkContainer : public std::enable_shared_from_this<ChunkContainer<data_t
     return matrix_bits_;
   }
 
+  int_t num_pow2_qubits(void)
+  {
+    return num_pow2_qubits_;
+  }
+  void update_pow2_qubits(void);
```

Comment:
Is `update_pow2_qubits()` called only by `DeviceChunkContainer:Allocate()`?

Reply:
Yes. I moved this to DeviceChunkContainer